libs/blocks/global-footer/global-footer.css

```diff
@@ -76,6 +76,8 @@
   flex-shrink: 0;
   font-size: 14px;
   font-weight: 700;
+  margin: 0;
+  line-height: 0;
```

Comment:
<img width="350" alt="Screenshot 2025-01-28 at 11 03 17" src="https://github.com/user-attachments/assets/ede1f2f7-4056-4999-8b6d-5e1ecf09e85d" />
Setting a `line-height` of 0 might be problematic.